{ 
  "text" : "Vixie Cron before the 3.0pl1-133 Debian package allows local users to cause a denial of service (use-after-free and daemon crash) because of a force_rescan_user error.\n" ,
  "entities" : [ { 
    "eid" : "E1" ,
    "keyid" : "Identity#6e9552c9bd8e61c8f277c21220160234" ,
    "type" : "Identity" ,
    "subType" : "Organization" ,
    "value" : "debian" ,
    "begin" : 33 ,
    "end" : 39 ,
    "canonicalForm" : "Debian" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E2" ,
    "keyid" : "TTP#9c98ef6a7451d6f82ca2194ee23d1acb" ,
    "type" : "TTP" ,
    "subType" : "TTP" ,
    "value" : "denial of service" ,
    "begin" : 78 ,
    "end" : 95 ,
    "canonicalForm" : "denial of service" ,
    "score" : "0.0"
  }
  , { 
    "eid" : "E3" ,
    "keyid" : "TTP#61dc059bc66336171fa7bd29101eb5cd" ,
    "type" : "TTP" ,
    "subType" : "TTP" ,
    "value" : "use-after-free" ,
    "begin" : 97 ,
    "end" : 111 ,
    "canonicalForm" : "use-after-free" ,
    "score" : "0.0"
  }
  ,  ] ,
  "relations" : [  ]
}
